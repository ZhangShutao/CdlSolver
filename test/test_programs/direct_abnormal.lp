% Program Reading Example
% WA 1: Unsat
% WA 2: <{a, -b},{a, -b}>

a :- &c{b}.
-b.