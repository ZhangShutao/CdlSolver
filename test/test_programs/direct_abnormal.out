default model: 1
x:-b; y:-b;
Satisfiable